orders/migrations/0003_static_orders.py

```diff
@@ -0,0 +1,69 @@
+from django.db import migrations, transaction
+from django.utils import timezone
+import datetime
+
+DOG_OWNER_ID = ['4', '5', '6', '7', '8', '9', '10', '11', '12', '13',
+                '14', '15', '16', '17', '18', '19', '20', '21', '22', '23',
+                '4', '5', '6', '7', '8', '9', '10', '11', '12', '13',
+                '14', '15', '16', '17', '18', '19', '20', '21', '22', '23']
+
+DAYCARE_ID = ['4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21',
+              '22', '23', '23', '22', '21', '20', '19', '18', '17', '16', '15', '14', '13', '12', '11', '10', '9', '8',
+              '7', '6', '5', '4']
+
+START_DATE = [timezone.datetime(2022, 3, 23), timezone.datetime(2022, 5, 23), timezone.datetime(2022, 2, 23),
```

Comment:
I think having it in tuple with the end date will be more readable for example:
`[(timezone.datetime(2022, 3, 23), 7), (timezone.datetime(2022, 5, 23),2)]`

Reply:
Great idea! Changed.